<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtraDataDialog</class>
 <widget class="QDialog" name="ExtraDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Простые стили для Qt Designer */

/* Все виджеты */
* {
    font-family: &quot;Segoe UI&quot;;
    font-size: 14px;
}

/* QLabel */
QLabel {
    color: #000000;
    font-weight: normal;
}

/* QLineEdit */
QLineEdit {
    border: 2px solid #CCCCCC;
    border-radius: 5px;
    padding: 5px;
    background: white;
}

QLineEdit:focus {
    border-color: #0066CC;
}

/* QSpinBox */
QSpinBox {
    border: 2px solid #CCCCCC;
    border-radius: 5px;
    padding: 5px;
    background: white;
}

/* QPushButton */
QPushButton {
    background-color: #0066CC;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #0055AA;
}

QPushButton:pressed {
    background-color: #004488;
}

QPushButton:disabled {
    background-color: #CCCCCC;
    color: #888888;
}

/* QDialog */
QDialog {
    background-color: #F5F5F5;
}

/* QCheckBox - ВАЖНО: минимальный стиль для отображения */
QCheckBox {
    spacing: 5px;
    color: #000000;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
}

QCheckBox::indicator:unchecked {
    border: 2px solid #888888;
    background-color: white;
    border-radius: 3px;
}

QCheckBox::indicator:checked {
    border: 2px solid #0066CC;
    background-color: #0066CC;
    border-radius: 3px;
    image: url();
}

</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>838</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEditGetAtesstat">
      <property name="placeholderText">
       <string>Выданный</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditSNILS">
      <property name="placeholderText">
       <string>СНИЛС</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditMother">
      <property name="placeholderText">
       <string>Мать</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelYear_2">
      <property name="text">
       <string>Год окончания</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEditSerialAttestat">
      <property name="placeholderText">
       <string>Серия Аттестата</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelParents">
      <property name="text">
       <string>Сведения о родителях(Фамилия, имя, отчество, почтовый адрес, место работы и должность)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditFather">
      <property name="placeholderText">
       <string>Отец</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="spinBoxYearGraduationAttestat">
      <property name="minimum">
       <number>2025</number>
      </property>
      <property name="maximum">
       <number>2976</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBoxYearDiploma">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>2025</number>
      </property>
      <property name="maximum">
       <number>2976</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="checkBoxPreviousPlace">
      <property name="text">
       <string>Предыдущее место образования</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditGetDiploma">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Выданный</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBoxDiploma">
      <property name="text">
       <string>Медаль (диплом &quot;с отличием&quot;)</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelYear">
      <property name="text">
       <string>Год окончания</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditSerialDiploma">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Серия Диплома</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditIDExtra">
      <property name="placeholderText">
       <string>Уникальный ID</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBoxAttestat">
      <property name="text">
       <string>Медаль (аттестат &quot;с отличием&quot;)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEditAdressEmail">
      <property name="placeholderText">
       <string>Адрес почты</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEditPhone">
      <property name="placeholderText">
       <string>НОМЕР ТЕЛЕФОНА</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExtraDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExtraDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
